lib/nsq-cluster/nsqd.rb

```diff
@@ -10,17 +10,20 @@ class Nsqd < ProcessWrapper
 
 
   def initialize(opts = {})
-    @host = opts[:host] || '127.0.0.1'
-    @tcp_port = opts[:tcp_port] || 4150
-    @http_port = opts[:http_port] || 4151
-    @lookupd = opts[:nsqlookupd] || []
-    @msg_timeout = opts[:msg_timeout] || '60s'
```

Comment:
@bschwartz @freerobby - I'm newer to the project, and don't quite see it. :smile: Why did you get rid of `msg_timeout`?

I ask because I was working on an unrelated PR to nsq-ruby and found this test failing on master, presumably because `msg_timeout` was removed here. - https://github.com/wistia/nsq-ruby/blob/master/spec/lib/nsq/consumer_spec.rb#L151